### ISSUE TITLE
Downlink Example ACK

### ISSUE THREAD

**Issue by BlairAllegroTech** [nicolsc/sigfox-downlink#6]
Would you be able to add to the downlink example an example (preferably the NodeJS example) of the expected Downlink ACK response.

I have an Oyster device which takes several configuration messages.
I am expecting an ACK to come back with  ```ack:true``` after sending a downlink message,  before i send the next configuration downlink message. 

Then finally when all configurations are sent i would respond to the last ACK with a 204 (No Content) do indicate that there are no more configuration messages ?

I could have completely misunderstood the documentation but an example would be really help full!!

Much Appreciated

**Comment by nicolsc** [comment]
Hi @BlairAllegroTech 

In fact it seems you would need to build your own applicative acknowledgement mechanism to achieve this.

By default, what happens is:
* Device A sends an uplink message, with the `ack` flag set to true. It will switch to RX mode a few seconds after transmission, after emitting this message
* Sigfox backend receives the message, and forward the downlink request to your application server using the `DATA > BIDIR` callbacks (if configured that way)
* Your application server responds within 10 secs either with a `204 No Content` or a `200 OK` and the appropriate JSON body
* If your reply is `200` (+valid JSON) ; the Sigfox backend will allocate this downlink message to the appropriate base station
* This base station will emit this message, this emission will trigger the `SERVICE > ACK` callback
* The device will receive the message, and automatically send an `Out Of Band` System message. This OOB frame will not trigger any callback, and may not be visible to you as a customer depending on your access rights.

Long story short : The only "downlink delivery" callback that you can subscribe (SERVICE>ACK) will notify you that the message has been emitted, which doesn't mean for sure that it was received by your device.

In this extensive blog post, @disk91 gets into the details and suggest a way to set up your own "downlink reception confirmation" mechanism
https://www.disk91.com/2017/technology/sigfox/sigfox-downlink-in-details/
![Diapositive13](https://user-images.githubusercontent.com/526931/91148273-b3e9be80-e6b9-11ea-8931-bd036cc40d37.jpg)
